rm/rm-server/src/test/java/functionaltests/nodesource/TestSSHInfrastructureV2.java

```diff
@@ -179,7 +179,7 @@ public void testSSHInfrastructureV2WithRestartDownNodes() throws Exception {
         RMTHelper.log("Final checks on the scheduler state");
         s = resourceManager.getState();
         assertEquals(NB_NODES, s.getTotalNodesNumber());
-        assertEquals(NB_NODES, s.getFreeNodesNumber());
+        assertEquals(NB_NODES, s.getTotalAliveNodesNumber()); // check amount of all nodes that are not down
```

Comment:
The state _Pending_ does not exist for Nodes.

The method `getTotalAliveNodesNumber` returns the number of Nodes which are not _DOWN_:

https://github.com/ow2-proactive/scheduling/blob/master/rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java#L699-L699